Resources/Private/JavaScript/Host/Service/TYPO3CR/NodeTreeService.js

```diff
@@ -88,7 +91,11 @@ class NodeTreeService {
                 })
             })
             .then(response => response.json())
-            .then(json => this.store.dispatch(actions.UI.PageTree.setSubTree(storePath, json)));
+            .then(json => {
+                const nodeWhichIsNotLoadingAnymore = $set(updatedRootInStore, 'isLoading', false);
+                this.updateNode(nodeWhichIsNotLoadingAnymore);
+                this.store.dispatch(actions.UI.PageTree.setSubTree(storePath, json));
+            });
```

Comment:
for this part, I'd be curious about a code review, specifically:

- is this code idiomatic?
- is the name "nodeWhichIsNotLoadingAnymore" good? (I think not). Better suggestions?